src/engraving/libmscore/excerpt.h

```diff
@@ -20,84 +20,79 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#ifndef __EXCERPT_H__
-#define __EXCERPT_H__
+#ifndef MU_ENGRAVING_EXCERPT_H
+#define MU_ENGRAVING_EXCERPT_H
 
 #include <QMultiMap>
 
 #include "types/fraction.h"
 #include "mscore.h"
 
 namespace Ms {
+class EngravingItem;
 class MasterScore;
-class Score;
 class Part;
-class Measure;
-class XmlWriter;
+class Score;
 class Staff;
 class XmlReader;
-class EngravingItem;
-
-//---------------------------------------------------------
-//   @@ Excerpt
-//---------------------------------------------------------
 
-class Excerpt : public QObject
+class Excerpt
 {
-    MasterScore* _oscore;
-
-    Score* _partScore           { 0 };
-    QString _title;
-    QList<Part*> _parts;
-    QMultiMap<int, int> _tracks;
-
 public:
-    Excerpt(MasterScore* s = 0) { _oscore = s; }
+    Excerpt(MasterScore* masterScore = nullptr) { m_masterScore = masterScore; }
     Excerpt(const Excerpt& ex, bool copyPartScore = true);
 
     ~Excerpt();
 
-    QList<Part*>& parts() { return _parts; }
-    const QList<Part*>& parts() const { return _parts; }
+    MasterScore* masterScore() const { return m_masterScore; }
+    Score* excerptScore() const { return m_excerptScore; }
+    void setExcerptScore(Score* s);
+
+    QString name() const { return m_name; }
+    void setName(const QString& title) { m_name = title; }
+
+    QList<Part*>& parts() { return m_parts; }
+    const QList<Part*>& parts() const { return m_parts; }
+    void setParts(const QList<Part*>& parts) { m_parts = parts; }
+
     bool containsPart(const Part* part) const;
 
     void removePart(const ID& id);
 
-    void setParts(const QList<Part*>& p) { _parts = p; }
-
     int nstaves() const;
     bool isEmpty() const;
 
-    QMultiMap<int, int>& tracks() { return _tracks; }
-    void setTracks(const QMultiMap<int, int>& tracks);
-
-    MasterScore* oscore() const { return _oscore; }
-    Score* partScore() const { return _partScore; }
-    void setPartScore(Score* s);
+    QMultiMap<int, int>& tracksMapping() { return m_tracksMapping; }
```

Comment:
Renaming `tracks()` into `tracksMapping()` seems to be an unrelated change and should be a separate change?

Reply:
Same for changing other methods in `excerps.h`/`excerpts.cpp`, like `oscore()` and `partScore()`.

Reply:
Yes, you're right, I should better split that commit into a separate PR.